lib/wallet/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -56,6 +56,8 @@ module Cardano.Wallet.Shelley.Transaction
     , mkTxSkeleton
     , mkUnsignedTx
     , txConstraints
+    , estimateNumberOfWitnesses
```

Comment:
Since this function returns `KeyWitnessCount`, why not name it as follows (it's also a bit more concise, with no loss of clarity):

```suggestion
    , estimateWitnessCount
```